pkg/virtctl/imageupload/imageupload.go

```diff
@@ -46,11 +46,13 @@ import (
 
 	cdiClientset "kubevirt.io/client-go/generated/containerized-data-importer/clientset/versioned"
 	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
```

Comment:
This is a redundant import.

Reply:
Oh my god I didn't see IDE warning about this one. Thanks Felix!